### PR TITLE
Update serverlessFunctions.md

### PR DESCRIPTION
**Pull request by noobling**
The sample code didn't seem to work for me. I've updated it to an example that did work.

### CODE REVIEW

**Comment by Tobbe** [comment]
Hi @noobling 
Where did you test your serverless function? Where didn't it work?
Netlify still say they support the callback format (The docs here https://docs.netlify.com/functions/build-with-javascript/#synchronous-function-format links to this forum topic that has the callback syntax as an example https://community.netlify.com/t/support-guide-how-do-i-write-a-javascript-lambda-function/24106)

**Comment by noobling** [comment]
Hey it was when I ran the code locally with rw dev. This was with the latest versions of redwood

**Comment by dthyresson** [comment]
@Tobbe and @noobling I think the change is welcome. 

But, should it not be async perhaps -- or at least suggest it might be, especially since it refers to using the Prisma db client which I believe will need an await?

The generators will create:

```
export const handler = async (event, context) => {
  return {
    statusCode: 200,
    body: JSON.stringify({
      data: '${name} function',
    }),
  }
}
```

Perhaps the docs should be:

```
```
export const handler = async (event, context) => {
  return {
    statusCode: 200,
    body: JSON.stringify({
      data: 'Serverless function',
    }),
  }
}
```

**Comment by noobling** [comment]
Sure updated

File: docs/serverlessFunctions.md [hunk 1]
```diff
@@ -11,11 +11,23 @@ at `http://localhost:8911/graphql`.
...
+    body: JSON.stringify({
+      data: 'Serverless function',
+    }),
+  }
 }
-```
\ No newline at end of file
+```
+
+You can also use the generator to create the function.
```
Comment (Tobbe) [inline docs/serverlessFunctions.md:29]:
```suggestion
Redwood comes with a generator to help you create serverless functions
```

**Comment by Tobbe** [comment]
Thanks for the PR @noobling

**Pull request merged**